packages/atlas/src/views/viewer/MemberView/ActivityItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react'
+
+import { Pill } from '@/components/Pill'
+import { Text } from '@/components/Text'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { imageUrlValidation } from '@/utils/asset'
+import { formatDateTime } from '@/utils/time'
+
+import {
+  ActivityItemContainer,
+  DateText,
+  DescriptionSkeletonLoader,
+  PillAndDateContainer,
+  PillSkeletonLoader,
+  Thumbnail,
+  ThumbnailSkeletonLoader,
+  Title,
+  TitleAndDescriptionContainer,
+  TitleSkeletonLoader,
+} from './ActivityItem.styles'
+
+export type ActivityItemProps = {
+  date: string
```

Comment:
I think this should be `Date` right? Our cache automatically wraps all ISO dates in `Date` type